### ISSUE TITLE
Missing library when publishing to Linux

### ISSUE THREAD

**Issue by DmitriZhukov** [jsakamoto/nupkg-selenium-webdriver-chromedriver#77]
Hi,

I have added package to Visual Studio, I set up environment variable in project
to copy the file to bin folder and its ok on Windows

`<PublishChromeDriver>true</PublishChromeDriver>`
`<DefineConstants>TRACE;_PUBLISH_CHROMEDRIVER</DefineConstants>`

When I compile and publish for Linux I have the following error

/app/chromedriver: error while loading shared libraries: libglib-2.0.so.0

Is it possible to include all libraries?

**Comment by jsakamoto** [comment]
> Is it possible to include all libraries?

No, it isn't possible.

It depends on shared library management policies in each Linux OS environment and depends on the architecture of the ChromeDriver.

So there is no way what I can do.

Who should responsible for maintaining shared library files on the OS that the application required is, Developers (or IT administrators), not package builders like me.

**Comment by jsakamoto** [comment]
See also: https://stackoverflow.com/questions/58134793/error-while-loading-shared-libraries-libnss3-so-while-running-gtlab-ci-job-to

**Comment by DmitriZhukov** [comment]
Thanks,
question then It asks for library
libX11-6
mind uppercase X and this cannot be found

How can I see list of all dependencie

**Comment by jsakamoto** [comment]
I'm not familiar with programming on Linux OS, so I had no idea.

Therefore, in the next step, I searched that error message `"chromedriver: error while loading shared libraries: libglib-2.0"` on Google.

Then I found the post on StackOverflow.com as below.

https://stackoverflow.com/questions/33382998/chromedriver-various-lib-dependencies-are-missing-on-ubuntu-14-04-64-bit

That thread referred to  `ldd` command.

So I tried `ldd` command in my WSL (Windows Subsystem for Linux) console, then I got the following list.

![image](https://user-images.githubusercontent.com/95908/98237367-20c9e080-1fa8-11eb-9994-634c97079311.png)

Is that helpful for you?
